packages/tex-to-myst/src/math.ts

```diff
@@ -37,4 +39,33 @@ export const MATH_HANDLERS: Record<string, Handler> = {
   'env_vmatrix*': mathBlockAsOriginal,
   env_Vmatrix: mathBlockAsOriginal,
   'env_Vmatrix*': mathBlockAsOriginal,
+  env_subequations(node, state) {
+    state.closeParagraph();
+    state.openNode('mathGroup');
+    state.renderChildren(node);
+    const mathGroup = state.top();
+    transformSubEquations(mathGroup);
+    state.closeNode();
+  },
 };
+
+function transformSubEquations(node: GenericNode) {
```

Comment:
I think this could be moved to a math transform in the future. Right now, the only thing that can create sub-equations is latex parsing, and this is targeted only at that!